applications/StructuralMechanicsApplication/custom_elements/total_lagrangian_mixed_volumetric_strain_element.h

```diff
@@ -0,0 +1,644 @@
+// KRATOS  ___|  |                   |                   |
+//       \___ \  __|  __| |   |  __| __| |   |  __| _` | |
+//             | |   |    |   | (    |   |   | |   (   | |
+//       _____/ \__|_|   \__,_|\___|\__|\__,_|_|  \__,_|_| MECHANICS
+//
+//  License:         BSD License
+//                   license: structural_mechanics_application/license.txt
+//
+//  Main authors:    Ruben Zorrilla
+//                   Riccardo Rossi
+//                   Guglielmo Scovazzi
+//
+
+#pragma once
+
+// System includes
+
+// External includes
+
+// Project includes
+#include "includes/define.h"
+#include "includes/element.h"
+#include "utilities/integration_utilities.h"
+
+// Application includes
+#include "structural_mechanics_application_variables.h"
+
+namespace Kratos
+{
+
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+
+///@}
+///@name  Enum's
+///@{
+
+///@}
+///@name  Functions
+///@{
+
+///@}
+///@name Kratos Classes
+///@{
+
+/**
+ * @class TotalLagrangianMixedVolumetricStrainElement
+ * @ingroup StructuralMechanicsApplication
+ * @brief Small displacement with strain based mixed formulation element
```

Comment:
```suggestion
 * @brief Finite strain with deformation gradient based mixed formulation element
```

Reply:
Copy & paste error. Thanks for spotting.

Reply:
Done. I took the chance to add better documentation.